src/model/format/reminder-dataview-plugin.ts

```diff
@@ -0,0 +1,196 @@
+import { DateTime } from "model/time";
+import moment from "moment";
+import type { Todo } from "./markdown";
+import { ReminderModel, TodoBasedReminderFormat } from "./reminder-base";
+import { escapeRegExpChars } from "./util";
+
+type DataviewSettingType = {
+    dateTrigger: string,
+    dateFormat: string,
+    timeTrigger: string,
+    timeFormat: string,
+    linkDateToDailyNote: boolean
+}
+
+const dataviewSetting = new (class DataviewSetting {
+
+    get dateTrigger() {
+        return this.get("date-trigger", "due:: ");
+    }
+
+    get dateFormat() {
+        return this.get("date-format", "YYYY-MM-DD");
+    }
+
+    get timeTrigger() {
+        return this.get("time-trigger", "time:: ");
+    }
+
+    get timeFormat() {
+        return this.get("time-format", "HH:mm");
+    }
+
+    get linkDateToDailyNote() {
+        return this.get("link-date-to-daily-note", false);
+    }
```

Comment:
I couldn't find these settings from Dataview plugin.
Are these settings correct?

Reply:
If there are no such settings in Dataview plugin, I feel we need these settings in obsidian-reminder plugin.

Reply:
`due` is part of the [shorthand syntax](https://blacksmithgu.github.io/obsidian-dataview/annotation/metadata-tasks/#field-shorthands) Dataview supports for tasks. I couldn't find a property built into Dataview specifically for tracking completion dates, but they use shorthand fields in their examples, so I think it's ok to include.

For `time`, there isn't a field like this in Dataview, but it's technically possible to track times using properties. I thought it would be better to include a time field as an option, but it might be better to wait until it's officially supported by the Tasks plugin first.